src/controllers/campaign.js

```diff
@@ -245,6 +257,16 @@ const getMarketingCampaignsById = async (req, res) => {
             });
         }
 
+        // Check if campaign is in flight
```

Comment:
@PedroLabrador this code is the same as it is in line 199 in this file, shouldn't we move it (abstract it) into a function so we enforce the rule of DRY (Don't Repeat Yourself)